spec.mediawiki

```diff
@@ -600,86 +579,6 @@ for a particular market proceeds as follows.
 An authenticated and authorized client may proceed to place orders as described 
 in Section 5.
 
-===Epoch Order Cycle===
```

Comment:
Temporarily deleted until the diagrams are updated?

Reply:
I'm not certain if I'll make diagrams or not. It is a bit simpler this way, and there was a ton of overlap between this and *Epoch-based Order Matching*, so I'm thinking it can all just be covered there. 

Do you see a benefit of reworking this section?

Reply:
Not presently, no.